dags/src/utility.py

```diff
@@ -279,7 +276,11 @@ def clip_soil_data(**kwargs):
         clipped_df = clipped_df.to_crs('EPSG:25832')
 
         # Write clipped output as .gpkg
-        output_path = os.path.join(current_dir, 'output', geo_tag, f'{geo_tag}_{date_now}.gpkg')
+        output_path = os.path.join(current_dir, 'output', geo_tag, 
```

Comment:
Please do not create a folder structure with parameters. Alternative: Create a .txt file with the same filename as metadata. It shall contain DAG-ID, execution start time, and given parameters like lat/lon.

Reply:
Oh ok - that's a great idea; I will make the adjustment now on this

Thanks